README.md

```diff
@@ -10,4 +10,32 @@
 * The game asks to enter players nickname before the start.
 * Card images are generated randomly
 * The image of the covered card is generated randomly (but it should look like the ordinary playing card back).
-* After revealed all the cards congratulations and the time the game took will pop up.
\ No newline at end of file
+* After revealed all the cards congratulations and the time the game took will pop up.
+
+## Development mode
+
+### Install project dependencies:
+
+`yarn` or `npm install`
+
+Installs all necessary dependencies.
+
+## Run the development server:
+
+`yarn start` or `npm run start`
+
+Runs the app in development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will automatically reload if you make changes to the code.
+
+### Build for production:
+
+`yarn build` or `npm run build`
+
+Builds the app for production to the `build` folder.<br>
+It correctly bundles React in production mode and optimizes the build for the best performance.
```

Comment:
Much text but little meaning. It would be more useful to write what to expect (what files it's creating more or less?) than this essay form.

Reply:
Thank you, I applied to your hints.

Reply:
A simple 👍  next to the comment can be enough